Automattic/sensei
# Add blocks from quiz template

Fixes #4595

### Changes proposed in this Pull Request

* Add remaining blocks to the editor (quiz actions, progress bar, back to lesson)
* Remove custom container blocks and form markup
* Add `form='sensei-quiz-form'` to all buttons in pagination and quiz actions
  * This makes sure they are part of the `<form>` that contains the questions, even if they are outside of it, like in the quiz template

### Testing instructions


* Enable the feature flag: `add_filter( 'sensei_feature_flag_course_theme, '__return_true' );`
* Open a quiz in learning mode, check that it works as expected
* Open the quiz template for editing (`Edit Site` in admin bar on the quiz page). Check that it looks mostly the same as the frontend
* Complete a quiz that has and one that doesn't have pagination enabled. Make sure Complete/Reset/Save buttons work


### Screenshot / Video

<img width="1527" alt="image" src="https://user-images.githubusercontent.com/176949/149846668-d18ef803-4c66-4405-b890-38f67d087978.png">
